docs/.vitepress/Footer.vue

```diff
@@ -25,32 +25,103 @@ const baseUrl = import.meta.env.BASE_URL;
   display: flex;
   justify-content: space-between;
   padding: 20px;
-  background-color: #f6f6f7;
+  background-color: #0F1728;
   width: 100%;
+  position: relative;
 }
-.has-sidebar ~ .footer {
-  width: calc(100% - 264px);
-  margin-left: 264px;
+@media (min-width: 960px) {
+  .has-sidebar ~ .footer {
+  width: calc(100% - var(--vp-sidebar-width, 300px));
+  margin-left: var(--vp-sidebar-width, 300px);
 }
+}
+@media (min-width: 1440px) {
+  .has-sidebar ~ .footer {
+    width: calc(100% - ((100% - (var(--vp-layout-max-width) - 64px)) / 2 + var(--vp-sidebar-width) - 32px));
+    margin-left: calc((100% - (var(--vp-layout-max-width) - 64px)) / 2 + var(--vp-sidebar-width) - 32px); 
+  }
+}
+
+.footer::before {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 104px;
+  background: url('stars-background.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  z-index: 1;
+  pointer-events: none;
+}
+
+.footer::after {
+  content: '';
+  position: absolute;
+  top: -8px;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 8px;
+  background: url('hero-wavy-line.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  transform: scaleY(-1);
+  z-index: 1;
+  pointer-events: none;
+}
+
 .footer-column {
   flex: 1;
   margin: 0 20px;
+  z-index: 2;
 }
+
 .footer-column-title {
   font-size: 14px;
   font-weight: 700;
+  color: #F6D553;
   text-transform: uppercase;
+  margin-bottom: 10px;
 }
+
 .footer-link {
   display: block;
   margin-bottom: 8px;
+  color: #FFFBD5;
+  font-size: 16px;
 }
 
 .footer-link:hover {
   text-decoration: underline;
 }
+
 .logo-column {
   display: flex;
   justify-content: flex-start;
 }
+
+@media (max-width: 960px) {
+  .footer {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+  }
```

Comment:
Might be nice to have this content aligned centrally on smaller screens, but I couldn't get it to work with flex-box properties 🤔 
<img width="492" alt="Screenshot 2024-09-23 at 08 29 17" src="https://github.com/user-attachments/assets/964021ab-5e5b-41fb-9714-9c703bfaf854">

Reply:
I just followed the [Figma-sketch](https://www.figma.com/design/yXE33bBCGrML9fG8s5LWTs/WARP-Portal?node-id=523-31891&node-type=frame&t=RHDWTWwJW92dy2nc-0) for the mobile version of the footer where the content is left-aligned :)